.github/workflows/linters.yml

```diff
@@ -0,0 +1,56 @@
+name: Linters
+
```

Comment:
-  [ ] It would be nice if you choose your branch name carefully. Because here you are using **first-feature** which is not so much descriptive. You could have used **feature/setup-hello-microverse** or **feature/initialize-hello-microverse** :+1: